test/e2e/config/docker.yaml

```diff
@@ -79,6 +79,7 @@ providers:
     - sourcePath: "../data/infrastructure-docker/v1alpha4/cluster-template-kcp-adoption.yaml"
     - sourcePath: "../data/infrastructure-docker/v1alpha4/cluster-template-machine-pool.yaml"
     - sourcePath: "../data/infrastructure-docker/v1alpha4/cluster-template-node-drain.yaml"
+    - sourcePath: "../data/infrastructure-docker/v1alpha4/cluster-template-md-and-machine-pool.yaml"
```

Comment:
I'm guessing this also needs to change to `cluster-template-upgrades.yaml`

Reply:
+1 to both